lib/platform/common.ts

```diff
@@ -1,6 +1,7 @@
 import got from 'got';
 import Git from 'simple-git/promise';
 import { RenovateConfig } from '../config/common';
+// eslint-disable-next-line import/no-cycle
```

Comment:
We want to avoid this. Maybe instead this should be exported from git/storage instead? Cc @ViceIce

Reply:
Yes, exactly. I was in doubt too.

Reply:
Also, keeping in mind that we are going to refactor a couple of functions, shouldn't we put these types in a specific place rather than declaring here and there?

Reply:
Ideally they should all be declared in a central location. But for ones that are pass-through to the git storage then we can store them there instead